README.md

```diff
@@ -5,6 +5,7 @@
 [![pypi version](https://img.shields.io/pypi/v/atomate2?color=blue)](https://pypi.org/project/atomate2)
 ![supported python versions](https://img.shields.io/pypi/pyversions/atomate2)
```

Comment:
Here is a second python badge still listing 3.9.

Reply:
that badge pulls the data it displays from PyPI, so will auto-update once this PR is merged and released

Reply:
Oki, good to know :)